src/desktop/apps/auction/components/__tests__/DOM.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import renderTestComponent from '../../__tests__/utils/renderTestComponent'
+import DOM from '../DOM'
+
+const rewire = require('rewire')('../Layout')
+const Layout = rewire.default
+
+xdescribe.only('DOM Interactions', () => {
```

Comment:
circle CI does not like this `xdescribe.only` (also generally, what's the state of these tests? Looks like they might be done but have some extra comments)

Reply:
These are the tests that I was never able to get working!